client/blocks/subscription-length-picker/option.jsx

```diff
@@ -0,0 +1,122 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import Badge from 'components/badge';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { localize } from 'i18n-calypso';
+import { TERM_ANNUALLY, TERM_BIENNIALLY, TERM_MONTHLY } from '../../lib/plans/constants';
+
+let componentNumber = 0;
```

Comment:
Do we really need to count exact instances? Try using lodash uniqueId. (internally this just increments a number.)

https://lodash.com/docs/4.17.5#uniqueId
```
uniqueId( 'subscription-option-' )
```

This is also necessary if we only ever expect _one_ of these on the page at a time.

Reply:
`uniqueID` seems like a good solution too. Is there any downside to the counting instances though?

Reply:
As a reader I usually question if there's some special need, versus generating a unique key.